apps/language_server/lib/language_server/build.ex

```diff
@@ -2,7 +2,9 @@ defmodule ElixirLS.LanguageServer.Build do
   alias ElixirLS.LanguageServer.{Server, JsonRpc, SourceFile, Diagnostics}
 
   def build(parent, root_path, opts) when is_binary(root_path) do
-    if Path.absname(File.cwd!()) != Path.absname(root_path) do
+    root_path = Path.expand(root_path)
```

Comment:
@fmterrorf I would not pollute this module with settings validation. Let's do it in server.ex in set_settings instead.

Reply:
Moved it to set_settings in this commit https://github.com/elixir-lsp/elixir-ls/pull/566/commits/a49ce6e89e0187f9f95c9f8f13e611ae96b336fc so that it'll use Path.expand instead of Path.absname